ModLoader/ModLoader.cs

```diff
@@ -177,7 +177,7 @@ public void LoadMod()
                 return;
             }
 
-            IEnumerable<Type> source = from t in selectedMod.assembly.GetTypes() where t.IsSubclassOf(typeof(VTOLMOD)) select t;
+            IEnumerable<Type> source = from t in Assembly.Load(File.ReadAllBytes(selectedMod.dllPath)).GetTypes() where t.IsSubclassOf(typeof(VTOLMOD)) select t;
```

Comment:
formatting could be a bit nicer

Reply:
`(from t in Assembly.Load(File.ReadAllBytes(selectedMod.dllPath)).GetTypes() 

where t.IsSubclassOf(typeof(VTOLMOD)) 

select t)`